### PR TITLE
Update CODEOWNERS - adding Jorge Garcia Hospital

### PR DESCRIPTION
**Pull request by bigludo7**
#### What type of PR is this?

Add one of the following kinds:

* subproject management

#### What this PR does / why we need it:

Add Jorge Garcia Hospital as code owner.

#### Which issue(s) this PR fixes:

Fixes #

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Comment by hdamker** [comment]
Added @jgarciahospital to @camaraproject/device-swap_codeowners, so the CODEOWNERS file will be valid after merge.

But shouldn't be @jgarciahospital also part of the @camaraproject/device-swap_maintainers then?

**Comment by bigludo7** [comment]
@hdamker Thanks
Just discussed with @jgarciahospital in our call --> Proposal to add Jorge as maintainer as well

**Review by hdamker — APPROVED** [review]

**Comment by hdamker** [comment]
@bigludo7 Thanks. Added @jgarciahospital also to @camaraproject/device-swap_maintainers now. So we are done here.

**Pull request merged**
